examples/README.md

```diff
@@ -0,0 +1,182 @@
+# Iceberg Java API Examples (with Spark)
+
+## About
+Welcome! :smile:
+
+If you've stumbled across this module, hopefully you're looking for some guidance on how to get started with the [Apache Iceberg](https://iceberg.apache.org/) table format. This set of classes collects code examples of how to use the Iceberg Java API with Spark, along with some extra detail here in the README.
+
+The examples are structured as JUnit tests that you can download and run locally if you want to mess around with Iceberg yourself. 
+
+## Running Iceberg yourself
+If you'd like to try out Iceberg in your own project, you can use the `spark-iceberg-runtime` dependency:
+```xml
+   <dependency>
+     <groupId>org.apache.iceberg</groupId>
+     <artifactId>iceberg-spark-runtime</artifactId>
+     <version>${iceberg.version}</version>
+   </dependency>
+```
+
+You'll also need `spark-sql`:
+```xml
+  <dependency> 
+    <groupId>org.apache.spark</groupId>
+    <artifactId>spark-sql_2.12</artifactId>
+    <version>2.4.4</version>
+  </dependency>
+```
+
+## Key features investigated
+The following section will break down the different areas of Iceberg explored in the examples, with links to the code and extra information that could be useful for new users. 
+
+### Writing data to tables
+There are multiple ways of creating tables with Iceberg, including using the Hive Metastore to keep track of tables (HiveCatalog), or using HDFS/ your local file system to store the tables. To limit complexity, these examples create tables on your local file system using the HadoopTables class.
+
+To write Iceberg tables you will need to use the Iceberg API to create a `Schema` and `PartitionSpec` which you use with a Spark `DataFrameWriter` to create Iceberg an `Table`.
+
+Code examples:
+- [Unpartitioned tables](src/test/java/WriteToUnpartitionedTableTest.java)
+- [Partitioned tables](src/test/java/WriteToPartitionedTableTest.java)
+
+#### A look quick look at file structures
+It could be interesting to note that when writing partitioned data, Iceberg will layout your files in a similar manner to Hive:
```

Comment:
We should add a warning here: you can't drop data into an Iceberg table and have it show up in your table because files are tracked individually, not by directory. Users should think of these directories like the storage underneath Postgres or MySQL: it is managed and can't be directly changed without going through Iceberg.

Reply:
Yes that's a good point, I'll add some extra clarification here